src/css/components/labels.less

```diff
@@ -57,16 +57,12 @@
 .label, .more {
```

Comment:
Actually my idea was to get rid of the label-class completely.
Before we already had only one class name -> .label.
Now you introduced .badge too without removing .label.
This raises complexity, which isn't the idea of this refactoring.

Too add specific styles on the labels-badge my idea was to use a selector like this:
.labels.badge